tutor-multi-chart/values.yaml

```diff
@@ -61,3 +61,19 @@ elasticsearch:
       xpack.security.transport.ssl.certificate: /usr/share/elasticsearch/config/certs/tls.crt
       xpack.security.transport.ssl.certificate_authorities: /usr/share/elasticsearch/config/certs/ca.crt
       xpack.security.transport.ssl.verification_mode: certificate
+
+# Configuration for the metrics server chart
+metricsserver:
+   # Control the chart inclusion
+   enabled: false
```

Comment:
I believe some other charts name this variable something like "installMetricsServer", though I'm not sure if that makes sense here.  Could that pattern be useful here?  Also, do we intend to enable people to link to an existing metrics server, but set this flag to false?

Reply:
@adzuci I took this `enabled` flag from the kube-prometheus-stack [Chart](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/Chart.yaml#L41). Could you please link an example of the pattern you're proposing?
Regarding the metrics-server option, in the docs we specify that this dependency should be included only if still not present in the cluster. However, we still need to test what happens when trying to install metrics-server if already present in a different namespace.

Reply:
The installation of the helm-chart will fail if metrics-server is present in a different namespace